test/support/api/actions.js

```diff
@@ -45,7 +45,7 @@ function generateData(patients = _.sample(this.fxPatients, 1)) {
 
   return {
     data,
-    included: [],
```

Comment:
I don't think this was breaking any tests, but was causing the patient name to be empty on actions:

![Screenshot 2023-03-28 at 1 27 30 PM](https://user-images.githubusercontent.com/35355575/228357901-270ee94b-b4a4-4e55-95f2-97966f7442b9.png)

I made the same change in the `test/support/api/flows.js` file, which was causing the same issue but for flows.